core/src/main/java/com/yahoo/ycsb/measurements/OneMeasurementHdrHistogram.java

```diff
@@ -101,8 +113,10 @@ public void exportMeasurements(MeasurementsExporter exporter) throws IOException
     exporter.write(getName(), "AverageLatency(us)", totalHistogram.getMean());
     exporter.write(getName(), "MinLatency(us)", totalHistogram.getMinValue());
     exporter.write(getName(), "MaxLatency(us)", totalHistogram.getMaxValue());
-    exporter.write(getName(), "95thPercentileLatency(us)", totalHistogram.getValueAtPercentile(95));
-    exporter.write(getName(), "99thPercentileLatency(us)", totalHistogram.getValueAtPercentile(99));
+
+    for (Integer percentile: percentiles) {
+      exporter.write(getName(), percentile + "thPercentileLatency(us)", totalHistogram.getValueAtPercentile(percentile));
```

Comment:
Very much nit: maybe change the label here? This will be grammatically incorrect for, say, the 91st percentile.